### ISSUE TITLE
Publish to .xlsm isn't working

### ISSUE THREAD

**Issue by sywldb** [chelh/VBASync#15]
Hello Chelsea!

First of all, great idea with VBASync, and thanks for sharing it with us! :)

## My Issue
I recently downloaded and compiled VBASync under VS2015 -- I'm not using C# 7.0, so had to make some changes around properties with "=>" and TryParse(..., out var s). My changes are purely around the syntax of C# 7.0, so no scope or logic has been changed.

I'm able to extract out VBA code from a test spreadsheet (simple HelloWorld function in a module), but not able to push back any changes -- I can see that the timestamp of the spreadsheet changes, so VBASync is writing changes to the spreadsheet. Clicking on the Refresh button on the VBASync gui, will show that there are changes to the module.

I've attached the solution with my changes.
[vbasync.zip](https://github.com/chelh/VBASync/files/919228/vbasync.zip)

Can you take a look to see what's going on?

Many thanks!
simon

PS. Some unit and integration tests would be great

**Comment by chelh** [comment]
Please try with the latest pre-built release candidate: [Portable.VBA.Sync.Tool.1.3.0.zip](https://github.com/chelh/VBASync/files/907981/Portable.VBA.Sync.Tool.1.3.0.zip)

**Comment by princekream** [comment]
Hi Chelsea,

This tool is such a great addition to the Office suite - well done!  I use it religiously for all my Excel VBA projects.

However, since I've started using it (v1.2 and now v1.3), I haven't been able to push changes to my Excel sheets.  Extracting works wonders, but not pushing.  As stated above by Simon, when pushing, the file datestamp does change so it looks like something is being written/changed, just not the code.

I tried 2 test cases just to confirm:
1) pushing a VBA (.bas) file on a sheet without a pre-existing VBA module
2) pushing a VBA (.bas) file on a sheet which had the VBA module already

I hope the above helps.  I've also tried on different computers, same result.  I confirm I have the latest .NET version also.

Thanks again.
